apis/nucleus/src/components/Cell.jsx

```diff
@@ -297,6 +297,19 @@ const Cell = forwardRef(
       },
     });
 
+    let bgColor = 'transparent';
+
+    if (
+      layout &&
+      layout.components &&
+      layout.components.find((comp) => comp.key === 'general') &&
+      layout.components.find((comp) => comp.key === 'general').bgColor &&
+      layout.components.find((comp) => comp.key === 'general').bgColor.color &&
+      layout.components.find((comp) => comp.key === 'general').bgColor.color.color
+    ) {
+      bgColor = layout.components.find((comp) => comp.key === 'general').bgColor.color.color;
+    }
```

Comment:
Also, don't you need to go through the theme to resolve the color?
Some thing like:
halo.public.theme.getColorPickerColor(bgColor.color)
I assume this is done in the client, but in a slightly different way?